lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -255,7 +255,8 @@ def self.get_or_create_app(params)
           Android: %w[Java React-Native Xamarin],
           iOS: %w[Objective-C-Swift React-Native Xamarin],
           macOS: %w[Objective-C-Swift],
-          Windows: %w[UWP WPF WinForms Unity]
+          Windows: %w[UWP WPF WinForms Unity],
+          Custom: %w[]
```

Comment:
please add custom in the platform too.
`Custom: %w[Custom]`

Because, without passing the platform as Custom, the Api throws error

`{
  "error": {
    "code": "BadRequest",
    "message": "Validation error for body parameter \"app.platform\": Missing required property: platform"
  }
}`